test/parallel/test-stdout-buffer-flush-on-exit.js

```diff
@@ -15,13 +15,11 @@ if (process.argv[2] === 'child') {
   process.exit();
 }
 
-[22, 21, 20, 19, 18, 17, 16, 16, 17, 18, 19, 20, 21, 22].forEach((exponent) => {
+[22, 16].forEach((exponent) => {
```

Comment:
What I'm trying to is that if a bug is introduced at a later date such that stdout is not flushed, the best thing is if the test will fail 100% of the time in that situation. With the change in this PR, if such a bug is introduced at a later date, it will not fail 100% of the time.

So, unless there's a compelling reason to make the change to the test, I would recommend against it. (However, there probably *is* a compelling reason, such as "If we don't make the change, the test times out on CI half the time." That's pretty compelling.)